internal/daysteps/daysteps.go

```diff
@@ -1,15 +1,40 @@
 package daysteps
 
 import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
 	"time"
+
+	"github.com/Yandex-Practicum/go1fl-4-sprint-final/internal/spentcalories"
 )
 
-var (
+const (
 	StepLength = 0.65 // длина шага в метрах
+	mKm        = 1000 // количество метров в километре.
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// ваш код ниже
+
+	res := strings.Split(data, ",")
+
+	if len(res) != 2 {
+		return 0, 0, errors.New("количество элементов слайса не равно 2")
+	}
+	steps, err := strconv.Atoi(res[0])
+	if err != nil {
+		return 0, 0, errors.New("невозможно преобразовать элемент слайса в int")
```

Comment:
1. Теряешь сообщение об ошибки
2. Ошибку лучше писать на английском
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```